cc.cubocore.CoreTime.yml

```diff
@@ -0,0 +1,44 @@
+app-id: cc.cubocore.CoreTime
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: coretime
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --share=network # Required to show weather info
+  - --socket=wayland
+  - --filesystem=home
+  - --filesystem=host
+  - --socket=pulseaudio # Required for ringing alarm
+  - --device=dri
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
notifications does not go through the portal?

Reply:
No, tested without it and it doesn't.